priv/monkey/_next_ls_private_compiler.ex

```diff
@@ -52,7 +52,13 @@ defmodule NextLSPrivate.Tracer do
 
   @source "user"
 
-  def trace(:start, _env) do
+  def trace(:start, env) do
+    Process.send(
+      parent_pid(),
+      {{:tracer, :start}, URI.parse(env.file).path},
```

Comment:
i'm fairly certain this is just a file path already, so you don't have to parse it as a URI and then convert it back to a path again

Reply:
You are right. I've just removed the unused `URI.parse` call